systemd-units/plymouth-poweroff.sh.in

```diff
@@ -2,4 +2,5 @@
 
 if [ "$1" = "poweroff" ]; then
 	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
+	sleep 1
```

Comment:
@carlocaione Why do we need this here? Will this make poweroff one second slower on any hardware?